dev-packages/node-integration-tests/package.json

```diff
@@ -27,9 +27,9 @@
   "dependencies": {
     "@aws-sdk/client-s3": "^3.552.0",
     "@hapi/hapi": "^21.3.10",
-    "@nestjs/common": "11.0.16",
-    "@nestjs/core": "10.4.6",
-    "@nestjs/platform-express": "10.4.6",
+    "@nestjs/common": "11.1.3",
```

Comment:
bumping & aligning to latest as that fixes some transitive dependency security issue.